### ISSUE TITLE
Ticket Holder - My Tickets / Ticket Transfer - Select Ticket(s) to Transfer 

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#218]
## Background
A user needs the ability to transfer tickets to other people. This will be done through the ticket Wallet. 

## Scenario: A User selects transfer ticket from the ticket view
- [ ] Given I have selected My tickets
- [ ] And Selected a Ticket
- [ ] When I select `Transfer Ticket`
- [ ] Then I am given the option to choose which tickets to transfer

### Mock 
<img width="336" alt="big_neon" src="https://user-images.githubusercontent.com/6877299/47678315-78e6d480-db8f-11e8-93a4-c91046fab152.png">

**Comment by Mcscott142** [comment]
@brittanygay should `218-ticket-xfer` branch should be ready for you. https://github.com/revelrylabs/bn-mobile-react/tree/218-ticket-xfer

**Comment by brittanygay** [comment]
Thanks @Mcscott142! Pulling this in to work on today.

**Comment by brittanygay** [comment]
**Update**:
I've gotten through most of the modal styling, and I'm working on the transfer card component right now. I'm hoping that the radio select styling will work out as planned, and I can override the default radio styles in React Native, but I'll holler if I come across any issues.

**Comment by brittanygay** [comment]
@zmelamed another suggestion in terms of saving time:

Instead of having a dashed line separating the ticket holder name and seating info, could we use a regular border? 

I'm doing some research to see which plugin is best to use (maybe this https://www.npmjs.com/package/react-native-dash) but I've also read that there are some bugs on Android. Just wanted to point this out because it could potentially save a lot of time!

**Comment by zmelamed** [comment]
To simplify just remove the seat info entirely, for the first 6 months we likely won’t have reserved seating so there won’t be any seat number,section, etc. just remove that row for now

**Comment by brittanygay** [comment]
Thanks @zmelamed!

**Update**:
I'm _almost_ done with everything, but I'm trying to make the modal scrollable, when there are multiple tickets shown. Once I finish that up I'll be able to get this into review.

**Comment by brittanygay** [comment]
Styles are now complete. More info and screenshots here: https://github.com/revelrylabs/bn-mobile-react/pull/241

**Passing over to @chas-mcmahon to help finish setting up the following**:
There's still some functionality that we need to clean up on the Radio buttons. I wasn't able to get the buttons to be un-selectable (but the selected state works)

**Comment by brittanygay** [comment]
React Native Radio Button that I used: https://www.npmjs.com/package/react-native-simple-radio-button

**Comment by chas-mcmahon** [comment]
I had to swap out the component since radios don't allow for multiple checked items and from what I can tell we need to allow transfer of several tickets at a time. I found a react native checkbox component that should work for our style needs.

Still ironing out the problem of getting the checkboxes to work w/ react state in a sane way, but making good progress. I'm not sure how to deal w/ submitting data in react native, but I'll talk to @daybreaker about it tomorrow once I've finished getting the checkboxes to toggle correctly.
